src/openrct2-ui/windows/TopToolbar.cpp

```diff
@@ -939,6 +939,25 @@ static void window_top_toolbar_paint(rct_window* w, rct_drawpixelinfo* dpi)
         gfx_draw_sprite(dpi, imgId, x, y, 3);
     }
 
+    // Draw map button overlay
+    if (window_top_toolbar_widgets[WIDX_MAP].type != WWT_EMPTY)
+    {
+        x = w->x + window_top_toolbar_widgets[WIDX_MAP].left + 4;
+        y = w->y + window_top_toolbar_widgets[WIDX_MAP].top + 1;
+        if (widget_is_pressed(w, WIDX_MAP))
+            y++;
+
+        static int32_t imageIdByRotation[] = {
```

Comment:
`const`

Reply:
I would even say this is constexpr. const can be still initialized during runtime, its just makes it immutable.